### ISSUE TITLE
Building error with erlang 22

### ISSUE THREAD

**Issue by saneery** [PSPDFKit-labs/bypass#108]
Error with the old version of cowlib
```
src/cow_multipart.erl:392: call to crypto:rand_bytes/1 will fail, since it was removed in 20.0; use crypto:strong_rand_bytes/1
make: *** [ebin/cowlib.app] Error 1
```

**Comment by ream88** [comment]
Is this fixed with `v2.1.0`?

**Comment by saneery** [comment]
> Is this fixed with `v2.1.0`?

Yes, it's fixed, thanks! :fire:

**Comment by ream88** [comment]
Thx a lot!
